modules/Adding-State-to-HTTP/exercises/Remember-Me-App/README.md

```diff
@@ -7,7 +7,7 @@ cookies to remember the name of visitors to our app.
 
 1. Create an empty directory & repository
 0. Run `npm init -y`
-0. Run `npm install --save express cookie-parser`
+0. Run `npm install --save express body-parser`
```

Comment:
I think if you asked people to read the cookie, and then do something with it, then you'd need cookie-parser (at least I was able to complete the specs without cookie-parser).